frontend/src/lib/components/PropertiesTable/PropertiesTimeline.tsx

```diff
@@ -0,0 +1,55 @@
+import { Properties } from '@posthog/plugin-scaffold'
```

Comment:
This file shouldn't exist anymore, since it was moved and refactored into `frontend/src/lib/components/PropertiesTimeline/PropertiesTimeline.tsx`. 😅 Removed it.